docs/api/cluster/silhouette.rst

```diff
@@ -7,29 +7,23 @@ The Silhouette Coefficient is used when the ground-truth about the dataset is un
 
 The Silhouette Visualizer displays the silhouette coefficient for each sample on a per-cluster basis, visualizing which clusters are dense and which are not. This is particularly useful for determining cluster imbalance, or for selecting a value for :math:`K` by comparing multiple visualizers.
 
-.. code:: python
+.. plot::
+    :context: close-figs
 
+    from sklearn.cluster import KMeans
     from sklearn.datasets import make_blobs
 
-    # Make 8 blobs dataset
-    X, y = make_blobs(centers=8)
-
-.. code:: python
-
-    from sklearn.cluster import MiniBatchKMeans
-
     from yellowbrick.cluster import SilhouetteVisualizer
 
+    # Generate synthetic dataset with 8 random clusters
+    X, y = make_blobs(n_samples=1000, n_features=12, centers=8, random_state=42)
+
     # Instantiate the clustering model and visualizer
-    model = MiniBatchKMeans(6)
+    model = KMeans(6)
```

Comment:
Using `KMeans` instead of `MiniBatchKMeans` didn't affect the plot so I changed it here to make it consistent with our other clustering visualizers. In addition, I was worried that new users might be confused about why it is used here and not in the other clustering examples.